handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -2141,6 +2142,10 @@ private void forceFlush(ChannelHandlerContext ctx) {
      */
     @Override
     public void channelActive(final ChannelHandlerContext ctx) throws Exception {
+        Channel c = ctx.channel();
+        if (c instanceof UnixChannel && engine instanceof ReferenceCountedOpenSslEngine) {
+            ((ReferenceCountedOpenSslEngine) engine).bioSetFd(((UnixChannel) c).fd().intValue());
+        }
```

Comment:
@normanmaurer @chrisvest should this be added to the `handlerAdded` method as well? When I posted about this PR in discord, someone mentioned that it's possible for a handler to be added after a channel is active.

Reply:
Yes!

Reply:
Added it in c74d81942d0b93f5e1b4794a28c45eb866150702 👍